config/path

```diff
@@ -45,6 +45,9 @@ TARGET_PREFIX=$TOOLCHAIN/bin/$TARGET_NAME-
 # use ARM toolchain on 64/32 split builds
 if [ -z "$KERNEL_TOOLCHAIN" -a "$TARGET_KERNEL_ARCH" = "arm64" -a "$TARGET_ARCH" = "arm" ]; then
   KERNEL_TOOLCHAIN="aarch64-none-linux-gnu"
+  if [ "${MACHINE_HARDWARE_NAME}" = "aarch64" ]; then
```

Comment:
nit: set KERNEL_TOOLCHAIN once. if x86_64, elif aarch64, else "No known toolchain available for ${MACHINE_HARDWARE_NAME}"